### DISCUSSION TITLE
Please, give me storage idea? 💡

### DISCUSSION THREAD

**Discussion by Obelixx** [microsoft/hack-together-teams#100] [Q&A] [answered]
I want to ask also for an idea, for a **fast and easy way** ⏩  to store data (a table + a keyword)? 💡
I need to store a simple table with 2-3 fields. It can be even text file. It will be good this data to be stored somewhere related to the company and not "in memory" or "on the hosting". 🏬  Furthermore, it will be good this table to be accessible from the teams/chat participants somehow. :accessibility: 
I wired my bot to the Graph API successfully. 💹 🌐 
Things that I consider are: 
* A file in Team chat, maybe?
* Or something with calendar?
* Or SharePoint folder and show it in a "tab app"? (but longer to implement (probably), considering the deadline 💀)

Please help! 🙏

**Comment by waldekmastykarz** [comment]
If you want to retrieve the data in a structured way, you could go with a SharePoint List. If you need just storage, you could create a file and store it either in a SharePoint site or in the user's OneDrive (choice depends on the use case). Does that give you enough pointers to continue?

**Reply by Obelixx** [reply]
The data is for the group chat and the bot - not so personal.
I have no experience with SharePoint, even as a user, so I'm a bit scared ... 👻

**Reply by waldekmastykarz** [reply, marked as answer]
You could check out our learn modules that focus on working with files using Microsoft Graph: here's one for JavaScript: https://learn.microsoft.com/en-us/training/modules/msgraph-manage-files/, and another one for C#: https://learn.microsoft.com/en-us/training/modules/msgraph-dotnet-core-manage-files/. I hope this helps

**Reply by Obelixx** [reply]
No time, will use in memory storage as is.
They say "Cloud is safe".... 🌩️
